.travis.yml

```diff
@@ -3,30 +3,57 @@ sudo: required
 dist: trusty
 
 go:
-  - "1.11.1"
+  - 1.11.x
+
+os:
+  - linux
+  - windows
 
 go_import_path: github.com/kubernetes-sigs/cri-tools
 
 services:
   - docker
 
 before_install:
-  - hack/install-docker.sh
-  - hack/install-kubelet.sh
+  - |
+    (
+      set -Eeuo pipefail
+      if [[ "$TRAVIS_OS_NAME" == "windows" ]]; then
+        choco install -y make mingw
+        export PATH="/c/tools/mingw64/bin:/c/ProgramData/chocolatey/bin;$PATH"
+      fi
+    )
 
 install:
   - make install.tools
 
 jobs:
   include:
     - stage: Static check
+      os: linux # Only need to run on Linux
       script:
         - make lint
         - make gofmt
     - stage: Build
+      os: linux # Only need to run on Linux
       script:
-        - make release
+        -  make release
     - stage: Test
+      os: linux
       script:
         - make
+        - hack/install-docker.sh
```

Comment:
kubelet installation moved here, so that static check and build could be run more quickly.